tendrl/ceph_integration/objects/ecprofile/atoms/create/__init__.py

```diff
@@ -25,6 +27,34 @@ def run(self):
                      plugin=plugin,
                      directory=directory
                      )
+        Event(
+            Message(
+                priority="info",
+                publisher=tendrl_ns.publisher_id,
+                payload={
+                    "message": "Creating ec-profile %s" %
+                    self.parameters['ECProfile.name'],
+                },
+                request_id=self.parameters['request_id'],
+                flow_id=self.parameters["flow_id"],
+                cluster_id=tendrl_ns.tendrl_context.integration_id,
+            )
+        )
+
         crud = Crud()
         crud.create("ec_profile", attrs)
+        Event(
+            Message(
+                priority="info",
+                publisher=tendrl_ns.publisher_id,
+                payload={
+                    "message": "Successfully created ec-profile %s" %
+                    self.parameters['ECProfile.name'],
+                },
+                request_id=self.parameters['request_id'],
+                flow_id=self.parameters["flow_id"],
+                cluster_id=tendrl_ns.tendrl_context.integration_id,
+            )
+        )
+
```

Comment:
What if there is a failure? How it is handled and logged?
Comment applicable for all CRUD atoms